swagger-ui/swagger.yaml

```diff
@@ -1696,6 +1700,436 @@
             description: Invalid query parameters
           500:
             description: Internal Server Error
+    /harvest/deposit:
+      post:
+        summary: Deposit funds to harvest liquidity pool
+        tags:
+          - harvest
```

Comment:
```suggestion
          - Harvest
```

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/670/commits/9dec028708bff2c2a439b41f591a736b518d3f06